subsys/bluetooth/services/fast_pair/fp_storage/include/fp_storage_pn.h

```diff
@@ -47,6 +47,12 @@ int fp_storage_pn_save(const char *pn_to_save);
  */
 int fp_storage_pn_get(char *buf);
 
+/** Clear storage data loaded to RAM.
+ *
+ * The function is used only by fp_storage unit test.
+ */
+void fp_storage_pn_ram_clear(void);
```

Comment:
why can't we have this one defined in the ```fp_storage_pn_priv.h```?

Reply:
We can do it. The thing is that we have similar case in account key storage - the `fp_storage_ak_ram_clear` function is already placed in `fp_storage_ak.h` and not in `fp_storage_ak_priv.h`. I would prefer to keep it consistent and move both `fp_storage_ak_ram_clear` and `fp_storage_pn_ram_clear` in their respective `_priv.h` files if that's ok with you and @MarekPieta.

Reply:
Moving the `_ram_clear` functions out of public API sounds good (tests could still rely on the private headers).